yak/CMakeLists.txt

```diff
@@ -147,6 +159,7 @@ target_include_directories(${PROJECT_NAME} PUBLIC
   ${PCL_INCLUDE_DIRS}
   ${EIGEN3_INCLUDE_DIR}
   ${OpenCV_INCLUDE_DIRS}
+  ${PROJECT_NAME}/include
```

Comment:
I'd expect this line to be redundant with lines [156 and 157](https://github.com/ros-industrial/yak/blob/7543e8a42585d5078d76a736f9f57e34ae65af71/yak/CMakeLists.txt#L156-L157), and in any case [`include_directories(include)` at line 86](https://github.com/ros-industrial/yak/blob/7543e8a42585d5078d76a736f9f57e34ae65af71/yak/CMakeLists.txt#L86) should do the same thing too.

Does it still build and run if we delete line 162?

Reply:
There may be redundant includes, but it would not compile until I added 
include_directories(include) with all the other stuff.